felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -2618,19 +2618,19 @@ func (m *bpfEndpointManager) ensureBPFDevices() error {
 	i := retries
 	for {
 		if err := netlink.NeighAdd(arp); err != nil && err != syscall.EEXIST {
-			log.WithError(err).Warnf("Failed to update neight for %s (arp %#v), retrying.", bpfOutDev, arp)
+			log.WithError(err).Warnf("Failed to update neigh for %s (arp %#v), retrying.", bpfOutDev, arp)
```

Comment:
I'm not remotely confident about this change. I _think_ that `Neigh` stands for `Neighbor` (or perhaps `Neighbour`). I couldn't find any particular reason for the `t`...

Reply:
It does.  Good change IMO.